src/app/dataValidator/rules/topstories.js

```diff
@@ -0,0 +1,55 @@
+import deepClone from '../../helpers/json/deepClone';
+import deepGet from '../../helpers/json/deepGet';
+
+const squashKeys = [
+  'responsive-top-stories',
+  'top-story',
+  'secondary-top-story',
+  'other-top-stories',
+];
+
+const squashTopStories = jsonRaw => {
+  const json = deepClone(jsonRaw); // make a copy so we don't corrupt the input
+  let groups = deepGet(['content', 'groups'], json);
+  let collectedItems = [];
+
+  if (groups) {
+    /*
+     * Find and delete unwanted groups while collecting items
+     */
+    groups = groups.filter(group => {
+      if (group.type && squashKeys.includes(group.type)) {
+        collectedItems = collectedItems.concat(group.items);
```

Comment:
```suggestion
       collectedItems.concat(group.items);
```

Reply:
> The concat() method is used to merge two or more arrays. This method does not change the existing arrays, but instead returns a new array.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat

Reply:
Fair enough, The screenshot above is confusion AF 😆